spec/rspec/core/example_spec.rb

```diff
@@ -90,7 +90,10 @@ def metadata_hash(*args)
       # ensure metadata is unique for each example
       expect(example.metadata.object_id).to_not eq(example2.metadata.object_id)
       expect(example.metadata[:custom_key]).to eq(nil)
+      expect(example.metadata[:block]).to_not eq(nil)
+
       expect(example2.metadata[:custom_key]).to eq(:custom_value)
+      expect(example2.metadata[:block]).to_not eq(nil)
```

Comment:
Instead of just asserting it is not nil, it would be good to make a positive assertion.  How about `expect(&example.metadata[:block]).to raise_error("first")`?